### PR TITLE
add switches for friendly descrition for name and location

### PR DESCRIPTION
**Pull request by vajonam**
Sorry I changed the variable names and set the defaults better. can you pull these in pls?

### CODE REVIEW

File: airprint/airprint-generate.py [hunk 1]
```diff
@@ -131,7 +132,11 @@ def generate(self):
                 tree.parse(StringIO(XML_TEMPLATE.replace('\n', '').replace('\r', '').replace('\t', '')))
 
                 name = tree.find('name')
-                name.text = 'AirPrint %s @ %%h' % (p)
+		
+                if self.descName:
+                   name.text = '%s' % (v['printer-info'])
+                else:
+                   name.text = 'Airprint %s @ %%h' % (p)
```
Comment (DrPsychick) [inline airprint/airprint-generate.py:139]: Please keep `AirPrint` with capital P as before

File: airprint/printer-update.sh [hunk 2]
```diff
@@ -1,10 +1,12 @@
 #!/usr/bin/env bash
 
-inotifywait -m -e close_write,moved_to,create /etc/cups |
+inotifywait -m -e close_write,moved_to,create /etc/cups &
```
Comment (DrPsychick) [inline airprint/printer-update.sh:3]: This does not work, the inotify  `|` must pass to the while loop

**Comment by vajonam** [comment]
the inotify should have & the, `|` does do anything it just waits there..

**Review by DrPsychick — CHANGES_REQUESTED** [review]
inotify command needs to be fixed

**Comment by vajonam** [comment]
k , got it my bad. I will clean this up.

**Comment by vajonam** [comment]
@DrPsychick , k sorted you have to delete the old description name. I think AVAHI_* is better name since its avahi stuff we are affecting.

File: airprint/airprint-generate.py (hunk 1 above)
Comment (DrPsychick) [reply airprint/airprint-generate.py]: please update this one last thing and we're good to merge

**Comment by vajonam** [comment]
@DrPsychick done.

File: airprint/airprint-generate.py (hunk 1 above)
Comment (vajonam) [reply airprint/airprint-generate.py]: done.

**Pull request merged**